test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -47,32 +48,47 @@ type StackManager struct {
 }
 
 type safeStackMap struct {
-	stacks map[string]*auto.Stack
-	lock   sync.RWMutex
+	stacks     map[string]*auto.Stack
+	loggers    map[string]io.Writer
```

Comment:
If we need to store, I'm not sure the loggers should dissociated from the stack. You could store everything together in a `struct`.

Reply:
I no longer store the logger, as you adviced!